Classes/View Controllers/UINavigationItem+TitleSubtitle.swift

```diff
@@ -11,10 +11,29 @@ import UIKit
 extension UINavigationItem {
 
     func configure(filePath: FilePath) {
-        configure(title: filePath.current, subtitle: filePath.basePath)
+        let accessibilityLabel: String?
+        switch (filePath.current, filePath.basePath) {
+        case (.some(let current), .some(let basePath)):
+            let isCurrentAFile = filePath.fileExtension != nil
+            let currentType = isCurrentAFile ? "File" : "Folder"
+            accessibilityLabel = .localizedStringWithFormat("%@: %@, file path: %@", currentType, current, basePath)
+        case (.some(let current), .none):
+            // NOTE: This is currently unused, as we early-exit
```

Comment:
Do we want to fix this / add support for this? Could create a `titleView` wrapping the title if an `accessibilityLabel` is also provided. Should create a separate ticket for that.